Source/Order.swift

```diff
@@ -125,6 +130,7 @@ final public class Order: NSObject, Codable {
         enum CodingKeys: String, CodingKey {
             case id
             case email = "email_sha256"
+            case isNew = "is_new"
```

Comment:
Will the behavior here be to exclude the key, or pass `null` as the value?

If the latter, we should make sure that the API is 👍  with that.

Reply:
It will exclude the key if not explicitly set. The API will be able to handle that, as that's what is currently being sent.

Reply:
Perfect — I couldn't remember what codable did with nil optionals 😊